ExeterTeam25/group-software-project
# Testing

To Test:
- pip install -r requirements
- MAKE SURE YOU HAVE CHROME INSTALLED ON YOUR COMPUTER
- run python manage.py test
- wait for tests to finish 
- there should be 18 tests that pass

Details:
- added testing to some areas of the project
